pkg/ir/ip.go

```diff
@@ -15,6 +15,10 @@ func IPFromString(s string) IP {
 	return IP{s}
 }
 
+func IPFromCidr(c CIDR) IP {
+	return IP{c.String()}
```

Comment:
What does it mean to convert a CIDR to IP? A CIDR represents a range of IPs.
The resulting string here may not be a legal IP